client/post-editor/editor-featured-image/index.jsx

```diff
@@ -122,7 +139,8 @@ class EditorFeaturedImage extends Component {
 		const { site, post } = this.props;
 		const featuredImageId = getFeaturedImageId( post );
 		const classes = classnames( 'editor-featured-image', {
-			'is-assigned': !! PostUtils.getFeaturedImageId( this.props.post )
+			'is-assigned': !! PostUtils.getFeaturedImageId( this.props.post ),
```

Comment:
I _think_ you can drop the `!!` here since it just does a "truthiness" check.

Reply:
Yeah, it works the same in my tests.